src/include/log.h

```diff
@@ -235,7 +235,7 @@ struct __wt_myslot {
     uint32_t flags;
 };
 
-#define WT_LOG_END_HEADER log->allocsize
+#define WT_LOG_END_HEADER 128
```

Comment:
I would prefer `#define WT_LOG_END_HEADER WT_LOG_ALIGN` over a second magic value of 128.